### DISCUSSION TITLE
Increase size of `remark` column

### DISCUSSION THREAD

**Discussion by rbovard** [QGEP/QGEP#761] [Ideas]
Currently, the `remark` columns are defined as `varchar(80)` wich is a little short.

Why not use a `text` type? Or if we want to keep the `varchar` type, at least increase its size?

Ping @ponceta

**Comment by sjib** [comment]
The remark field in the datamodel is limited to 80 characters as this is also the case in the VSA-DSS datamodel. If you need further documentation on an object please use the attributes specified for this. Or you can connect a document / foto / ... with the file class. Remark should not be missused to store attribute information that would need it's own proper attribute.

**Comment by urskaufmann** [comment]
Hint: In the default project all remark-fields are configured as multiline. That's wrong, because in VSA-DSS the remarks are defined as Text, not as MText. If you have linefeeds in the remark-field, you will get an error when exporting with INTERLIS. In the next default project, that Multiline has to be changed (I have already done).

**Comment by ponceta** [comment]
@rbovard If you don't expect to export your data to Interlis in the next 20 years, then :

```
ALTER TABLE qgep_od.wastewater_structure
    ALTER COLUMN remark TYPE text;
```

And regenerate all views.

To be saved as customization SQL, will impact all wastewater structures (including reaches).
